Tests/MSFT_xDhcpServerAuthorization.Tests.ps1

```diff
@@ -0,0 +1,134 @@
+[CmdletBinding()]
+param()
+
+if (!$PSScriptRoot) # $PSScriptRoot is not defined in 2.0
```

Comment:
I thought that too. However, I left it in when I copied the file from another MS resource that had some tests. I'm happy to remove it if that's the concensus?